components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -148,11 +148,10 @@ func (p *pg) ListPolicies(ctx context.Context) ([]*v2.Policy, error) {
 
 	var pols []*v2.Policy
 	rows, err := p.db.QueryContext(ctx,
-		`SELECT query_policies from query_policies($1);`, pq.Array(projectsFilter))
+		"SELECT query_policies from query_policies($1)", pq.Array(projectsFilter))
```

Comment:
Whoa I feel like we have backticks everywhere for these. . . .

Reply:
Totally doesn't matter. I just prefer to use backticks only where they're the most handy choice -- when there's `"` in the string I'd otherwise have to escape. If there's nothing to escape, I'd pick plain `"`. I've just done that because I was touching code near these anyways. It doesn't matter 😃